game-core/src/main/java/games/strategy/triplea/ui/UnitChooser.java

```diff
@@ -302,6 +302,10 @@ private void addToCollection(final Collection<Unit> addTo, final ChooserEntry en
     });
   }
 
+  public void addChangeListener(final ScrollableTextFieldListener listener) {
+    entries.stream().forEach(entry -> entry.addChangeListener(listener));
```

Comment:
`List` has a convenience `forEach` method attached to it. FYI can simplify a little bit with it:
```
entries.forEach(entry -> entry.addChangeListener(listener));
```